src/main/resources/openapi.json

```diff
@@ -613,24 +613,6 @@
                                 }
                             }
                         }
-                    },
-                    "404": {
```

Comment:
I asked to remove 404 from the admin endpoint when creating a topic (and you did that) but why removing the 404 when you GET a topic? it could not exist. Anyway, it's totally unrelated to this PR.

Reply:
I'm so sorry, it's totally my fault